CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+## Code of Conduct
+
+Please note that this project is released with a [Code of Conduct](./CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms.
+
+## Matrix room
+
+Joining the matrix room at [#codespan:matrix.org][codespan-matrix] is a good way to get in touch with the developers and community.
+
+[codespan-matrix]: https://app.element.io/#/room/#codespan:matrix.org
+
+## Workflow
+
+Follow these steps to contribute to the project:
+
+1. Make a fork of the [codespan repository][codespan-repo].
+1. Within your fork, create a branch for your contribution. Use a meaningful name.
+1. Create your contribution, meeting all [contribution quality standards](#quality-standards).
+1. Ensure all the tests pass (`cargo test`).
+1. [Create a pull request][create-a-pr] against the `master` branch of the repository.
+1. Once the pull request is reviewed and CI passes, it will be merged.
```

Comment:
Should collaborators merge their own PRs or should they still be reviewed by someone else?

Reply:
Hmm, I think in general it's ok for collaborators to merge them themselves if they are approaved. We could maybe look at adding Mergify or Bors support if this makes things easier?

Reply:
In other words, there are no other limitations than the last point and collaborators can merge themself if CI passes.

Reply:
Yeah, pretty much!

Reply:
I think an approved review is kind of like “I trust you to clean stuff up if I had any things I mentioned, but otherwise it looks good, merge when you're ready!”